sources/packages/api/src/app.aest.module.ts

```diff
@@ -92,6 +95,10 @@ import { ApplicationControllerService } from "./route-controllers/application/ap
     GCNotifyService,
     GCNotifyActionsService,
     AssessmentControllerService,
+    StudentControllerService,
+    StudentFileService,
```

Comment:
This has been already imported in line 70 of the providers, can we please remove it here?